packages/ag-charts-enterprise/src/features/annotations/axisButton.ts

```diff
@@ -0,0 +1,134 @@
+        this.button = button;
+        this.toggleVisibility(false);
+        this.updateButtonElement();
+
+        const seriesRegion = this.ctx.regionManager.getRegion(REGIONS.SERIES);
+
+        this.destroyFns.push(
+            seriesRegion.addListener('hover', (event) => this.onHover(event)),
+            seriesRegion.addListener('leave', () => this.onLeave()),
+            () => this.destroyElements(ctx.domManager),
+            () => this.wrapper.remove(),
+            () => this.button.remove()
+        );
+    }
+
+    private onHover(event: _ModuleSupport.PointerInteractionEvent<'hover'>) {
+        const { enabled } = this;
+        if (!enabled) return;
+
+        this.toggleVisibility(true);
+        const buttonCoords = this.getButtonCoordinates(event);
+        this.coords = this.getAxisCoordinates(buttonCoords);
+        this.updatePosition(buttonCoords, this.enabled);
+    }
+
+    private onLeave() {
+        this.toggleVisibility(false);
+    }
+
+    private getButtonCoordinates(event: _ModuleSupport.PointerInteractionEvent<'hover'>) {
+        const {
+            axisCtx: { direction, position },
+            seriesRect,
+        } = this;
+
+        const { clientWidth: buttonWidth, clientHeight: buttonHeight } = this.button;
+
+        let x = 0;
+        let y = 0;
+        if (direction === ChartAxisDirection.X) {
+            x = event.offsetX - buttonWidth / 2;
+            y = position === 'top' ? seriesRect.y : seriesRect.y + seriesRect.height - buttonHeight;
+        } else {
+            x = position === 'left' ? seriesRect.x : seriesRect.x + seriesRect.width - buttonWidth;
+            y = event.offsetY - buttonHeight / 2;
+        }
+
+        return { x, y };
+    }
+
+    private getAxisCoordinates(coords: Coords) {
+        const { seriesRect } = this;
+        const { clientWidth: buttonWidth, clientHeight: buttonHeight } = this.button;
+
+        const x = coords.x - seriesRect.x + buttonWidth / 2;
+        const y = coords.y - seriesRect.y + buttonHeight / 2;
+        return {
+            x,
+            y,
+        };
+    }
+
+    private toggleVisibility(visible: boolean) {
+        const { button } = this;
+        if (button == null) return;
+
+        const isVisible = this.enabled && visible;
+        this.toggleClass('hidden', !isVisible);
+    }
+
+    private toggleClass(name: string, include: boolean) {
+        this.wrapper.classList.toggle(`${DEFAULT_ANNOTATION_AXIS_BUTTON_CLASS}-${name}`, include);
+    }
+
+    private updatePosition({ x, y }: Coords, visible: boolean) {
+        if (!visible) {
+            return;
+        }
+
+        this.wrapper.style.transform = `translate(${Math.round(x)}px, ${Math.round(y)}px)`;
+    }
+
+    private updateButtonElement() {
+        const { button } = this;
+        button.onclick = _ModuleSupport.makeAccessibleClickListener(button, () => this.onButtonClick(this.coords));
+        button.innerHTML = `<span>+</span>`;
```

Comment:
We should use an icon instead of text:

e.g.

```css
/* community/src/styles/styles.css */

.ag-charts-icon-plus {
    mask-image: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyNCIgaGVpZ2h0PSIyNCIgdmlld0JveD0iMCAwIDI0IDI0IiBmaWxsPSJub25lIiBzdHJva2U9ImN1cnJlbnRDb2xvciIgc3Ryb2tlLXdpZHRoPSIyIiBzdHJva2UtbGluZWNhcD0icm91bmQiIHN0cm9rZS1saW5lam9pbj0icm91bmQiIGNsYXNzPSJsdWNpZGUgbHVjaWRlLXBsdXMiPjxwYXRoIGQ9Ik01IDEyaDE0Ii8+PHBhdGggZD0iTTEyIDV2MTQiLz48L3N2Zz4=);
}
```

```ts
button.classList.add('ag-charts-icon-plus');
```